packages/libs/react-ui/src/components/Footer/Footer.css.ts

```diff
@@ -0,0 +1,162 @@
+import { ColorType, sprinkles, vars } from '../../styles';
+
+import { style, styleVariants } from '@vanilla-extract/css';
+
+export const footerVariants = styleVariants({
+  web: [sprinkles({ width: '100%', maxWidth: '100%', flexDirection: 'row' })],
+  mobile: [
+    sprinkles({
+      width: '100%',
+      maxWidth: 'maxContent',
+      flexDirection: 'column',
+    }),
+  ],
+});
+
+export const containerClass = style([
```

Comment:
Could we merge `FooterPanel` and `Footer` into a single element?

Reply:
<img width="372" alt="Screenshot 2023-07-12 at 14 13 03" src="https://github.com/kadena-community/kadena.js/assets/132443209/628324a8-b4ae-4b28-b623-6cbee7132946">
<img width="938" alt="Screenshot 2023-07-12 at 14 13 50" src="https://github.com/kadena-community/kadena.js/assets/132443209/25b258cb-5fd1-4baa-b92e-b89826233dd0">

I created the panel subcomponent so we can have different parts of items wrapped together and can handle them together as a unit and use the Footer (the root) as a wrapper around those. @bearni95 Let me know what you think about this